.babelrc

```diff
@@ -1,7 +1,6 @@
 {
   "presets": [
-    ["es2015", { "modules": false }],
+    "es2015",
```

Comment:
FWIW, I don't think it's the end of the world if we lose tree shaking for the sake of consistency.